app/controllers/api/v1/accounts/featured_tags_controller.rb

```diff
@@ -17,6 +17,6 @@ def set_account
   end
 
   def set_featured_tags
-    @featured_tags = @account.suspended? ? [] : @account.featured_tags
+    @featured_tags = @account.unavailable? ? [] : @account.featured_tags
```

Comment:
Is this here on purpose, and/or related to the other changes?

Reply:
I just found it inconsistent with other uses when looking up how other controllers define this. It's actually an alias anyway.